pkg/pillar/scripts/device-steps.sh

```diff
@@ -433,7 +435,9 @@ else
 fi
 if [ ! -s "$DEVICE_CERT_NAME" ]; then
     echo "$(date -Ins -u) Generating a device key pair and self-signed cert (using TPM if available)"
-    mount -o remount,flush,dirsync,noatime,rw /config
+    mount -o remount,rw $CONFIGDIR
```

Comment:
Without noatime the access time might be recorded (it would be in a normal file systen; I'm not sure if FAT32 records the atime.) If it does, it means we'll do additional modifications to the metadata for no reason. Hence I'd keep noatime.

Reply:
remount changes only options specified on commanf line after -o

```
$mount | grep config
configfs on /sys/kernel/config type configfs (rw,nosuid,nodev,noexec,relatime)
tmpfs on /config type tmpfs (ro,nosuid,nodev,noexec,relatime,size=256k,mode=755)
tmpfs on /containers/onboot/005-measure-config/tmp type tmpfs (rw,relatime)

$mount -o remount,rw /config

$mount | grep config
configfs on /sys/kernel/config type configfs (rw,nosuid,nodev,noexec,relatime)
tmpfs on /config type tmpfs (rw,nosuid,nodev,noexec,relatime,size=256k,mode=755)
tmpfs on /containers/onboot/005-measure-config/tmp type tmpfs (rw,relatime)

$mount -o remount,ro /config

$mount | grep config
configfs on /sys/kernel/config type configfs (rw,nosuid,nodev,noexec,relatime)
tmpfs on /config type tmpfs (ro,nosuid,nodev,noexec,relatime,size=256k,mode=755)
tmpfs on /containers/onboot/005-measure-config/tmp type tmpfs (rw,relatime)
```

Reply:
Oh didn't know that.